lang/en/texts/sendinblue.html

```diff
@@ -129,7 +129,7 @@
           <div class="sib-form-block" style=" font-size:14px; text-align:center; font-family:&quot;Helvetica&quot;, sans-serif; color:#333; background-color:transparent; border-width:px;">
             <div class="sib-text-form-block">
               <p>
-                <a href="https://world.openfoodfacts.dev/legal" target="_blank">Terms and Privacy</a>
+                <a href="https://world.openfoodfacts.org/legal" target="_blank">Terms and Privacy</a>
```

Comment:
Is this HTML available on other domains?
```suggestion
                <a href="/legal" target="_blank">Terms and Privacy</a>
```